layouts/community/ortho_3x10/wanleg/config.h

```diff
@@ -0,0 +1,12 @@
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace these two lines with just `#pragma once`, and remove the endif at the bottom?